src/metrics/prometheus.ts

```diff
@@ -0,0 +1,38 @@
+import { collectDefaultMetrics, Counter, Gauge, register, Summary } from 'prom-client';
+
+import config from '../config';
+
+register.setDefaultLabels({ service: 'c2c_tracking' });
+
+// default metrics
+if (!process.env['JEST_WORKER_ID']) {
+  collectDefaultMetrics();
+}
+
+// API call errors counter
+export const promApiErrorsCounter = new Counter({
+  name: 'api_errors_counter',
+  help: 'API Errors Counter',
+  labelNames: ['vendor', 'name', 'code'],
+});
+
+// unhandled errors
+export const promUnhandledErrorsCounter = new Counter({
+  name: 'unhandled_errors_counter',
+  help: 'Unhandled Errors Counter',
+});
+
+// API response time
+export const promResponseTimeSummary = new Summary({
+  name: 'response_time',
```

Comment:
without tags that won't be very helpful (but maybe ok in your case)

you cannot distinguish between the different routes, methods etc. You will only have a global response time